tests/build/common_include_dir.srctree

```diff
@@ -16,18 +16,17 @@ PYTHON fake_grep.py -c '#include "common/AddTraceback_impl_.*h"' c.c
 
 import sys
 from Cython.Build.Dependencies import cythonize
-import platform
 import os
 
 from distutils.core import setup
 
-# os x on Travis specifically seems to crash with nthreads>0
-osx_on_travis = (platform.system() == "Darwin" and os.getenv("TRAVIS"))
+# os x on CI specifically seems to crash with nthreads>0
+osx_on_ci = (sys.platform == "darwin" and os.getenv("CI"))
 
 # Test concurrent safety if multiprocessing is available.
-# (In particular, TravisCI does not support spawning processes from tests.)
+# (In particular, CI does not support spawning processes from tests.)
```

Comment:
```suggestion
# (In particular, TravisCI does not support spawning processes from tests, and so may others.)
```

Reply:
I think the workaround was necessary on both Travis and Github Action.

So `(In particular, neither TravisCI not Github Actions support spawning processes from tests.)`?

Reply:
Works for me.